charts/influxdb/templates/elastisys_custom/influx-auth-secret.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
   name: {{ template "influxdb.fullname" . }}-auth
-data:
-  INFLUXDB_USER: {{ .Values.elastisys_custom.influxdb_user | b64enc }}
-  INFLUXDB_PWD: {{ .Values.elastisys_custom.influxdb_password | b64enc }}
+stringData:
+  INFLUXDB_ADMIN_USER: {{ .Values.elastisys_custom.influxdbAdminUser }}
+  INFLUXDB_ADMIN_PWD: {{ .Values.elastisys_custom.influxdbAdminPassword }}
```

Comment:
Does this name change affect anything else?

Reply:
Yes, it effects https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/influxdb.yaml.gotmpl#L46-L58